### DISCUSSION TITLE
Kali on ARM "MT8183"

### DISCUSSION THREAD

**Discussion by JamesEpps** [dnschneid/crouton#4570] [Show and tell]
Couldn't find a solution elsewhere, so I will just leave this here.

HP Chromebook 11a with MT8183 "jacuzzi":
I had to specify armel architecture. That was it. Armel is probably a better choice right now, either way.
"-a armel"

Hope this helps someone!

crouton: version 1-20210709170820~master:14ca6bc7
release: kali-rolling
architecture: armel
xmethod: xorg
targets: audio,core,extension,keyboard,xorg,xiwi,x11,xfce
host: version 14092.28.0 (Official Build) beta-channel jacuzzi 
kernel: Linux localhost 4.19.197-12077-g23a4fd0addc4 #1 SMP PREEMPT Tue Aug 10 21:57:50 PDT 2021 aarch64 GNU/Linux

**Comment by n8-mills** [comment]
This may help me, I've been struggling with finding a distro that will install, run, and allow for installation of 64 bit apps.  I have the same processor in my new Acer. Will try later.

**Reply by JamesEpps** [reply]
You will be running "32bit" ARM within a 64bit kernel. We are needing arm64 repos for that. x64 PC architecture stuff is even further off, and would all be through emulation. Mine still works great for anything armel (arm 32bit), and I don't know if it's the 8 cores or what but it is FAST! Updating through the crouton script was broken for me last time I tried, but I typically "apt full-upgrade", which works anyway. Good Luck!

**Reply by n8-mills** [reply]
Ah I see, so there's no 64 bit support in Crouton?  Or just Kali?

**Reply by JamesEpps** [reply]
For this machine, I only needed Kali, so I haven't tried. Debian should have support first, then try Ubuntu. You may have to list out the arches with the crouton script then manually specify, ex. "-a armhf", "-a aarch64", or maybe "-a arm64". Give it a shot!
